packages/peregrine/lib/talons/CheckoutPage/__tests__/useCheckoutPage.spec.js

```diff
@@ -0,0 +1,209 @@
+import React from 'react';
```

Comment:
Thank you @revanth0212 for adding this test coverage 🙏 was just about to do the same in scope of my PR, glad I took a peek at this.